src/olympia/ratings/admin.py

```diff
@@ -0,0 +1,14 @@
+from django.contrib import admin
+
+from .models import Rating
+
+
+class RatingAdmin(admin.ModelAdmin):
+    raw_id_fields = ('addon', 'version', 'user', 'reply_to',)
+    readonly_fields = ('addon', 'version', 'user', 'reply_to', 'ip_address',
+                       'body', 'rating')
```

Comment:
it's more of a nit, but I find specifying which fields are readonly, rather than which fields are writeable, an annoying opaque way of doing it.  In Serializers I took to doing `read_only_fields = tuple(set(fields) - set(writeable_fields))`